package.json

```diff
@@ -6,12 +6,16 @@
   "author": "Nikhil Arora <narora200@gmail.com>, Saumya Tiwari <saumyatiwari.29@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "start:dev": "echo \"starting the-remotants backend\" && nodemon src",
+    "start:dev": "nodemon src --exec babel-node",
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "test:db": "nodemon src/testDatabase.js --exec babel-node"
```

Comment:
what is this for?

Reply:
Used it for exclusively running testDatabase.js for testing purpose. Removed now.